website/site/content/docs/configuration-options.md

```diff
@@ -60,6 +60,19 @@ public_folder: "/images/uploads"
 
 Based on the settings above, if a user used an image widget field called `avatar` to upload and select an image called `philosoraptor.png`, the image would be saved to the repository at `/static/images/uploads/philosoraptor.png`, and the `avatar` field for the file would be set to `/images/uploads/philosoraptor.png`.
 
+## Slug Type
+
+By default, filenames (slugs) for entries created in the CMS are sanitized according to [RFC3987](https://tools.ietf.org/html/rfc3987) and the [WHATWG URL spec](https://url.spec.whatwg.org/). This spec allows non-ASCII (or non-Latin) characters to exist in URLs. However, for maximum compatibility, you can also set a different slugification option:
+
+- `iri` (default): Keeps Unicode characters in slugs, according the the IRI draft spec ([RFC3987](https://tools.ietf.org/html/rfc3987)) and the [WHATWG URL spec](https://url.spec.whatwg.org/).
+- `latin`: Removes accents/diacritics from slug, then strips out all non-valid URL characters and periods (see `ascii` below).
+- `ascii`: Strips out all characters except valid URI chars (RFC3986) or periods (0-9, a-z, A-Z, `_`, `-`, `~`).
+
+**Example**
+
+``` yaml
+slug_type: "latin"
```

Comment:
`slug_type` is a bit general, and could be confusing when we finally do more with slugs and potentially have more configuration options for slugs. What do you think about `slug_characters` or `slug_chars`?

Reply:
Maybe `slug_charset`, or is that too much overlap with HTTP/HTML type stuff?

Reply:
`slug-encoding`?

Reply:
Any of the above could work, I shied away from really specific terms like encoding because I'm not sure it's technically accurate, and yeah "charset" kind of has a specific meaning too. At any rate, just something accurate that isn't as broad as "type".

Reply:
Jekyll uses filter: i.e. `slug_filter`.
@erquhart I'm fine with whatever you like, I really don't have a hard preference.

Reply:
I see your worry about technical accuracy, but with the prevalence across languages of terms like `url_encoded`, I think `slug-encoding`/`slug_encoding` feels most naturally understandable.
(It also would be nice if we consistently knew whether to use `-` or `_`, but it's too late for that! 😝)

Reply:
The convention used in Netlify CMS is underscore.

Encoding sounds great, I'm with that.

Reply:
> The convention used in Netlify CMS is underscore.

...except when it's hyphen, like `yaml-frontmatter`, or camel case, like `valueField`. ;)

Underscore is certainly the most common, though, and good to know that's the standard going forward. (I wonder if that makes sense to document in the contributor docs?)

Reply:
I think the options will be `unicode` and `ascii`  now. @verythorough @erquhart Do you think `slug: encoding` still makes sense for that?